Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -61,21 +61,18 @@ NativeReanimatedModule::NativeReanimatedModule(std::shared_ptr<CallInvoker> jsIn
                                                std::unique_ptr<jsi::Runtime> rt,
                                                std::shared_ptr<ErrorHandler> errorHandler,
                                                std::function<jsi::Value(jsi::Runtime &, const int, const jsi::String &)> propObtainer,
-                                               PlatformDepMethodsHolder platformDepMethodsHolder) : NativeReanimatedModuleSpec(jsInvoker),
-                                                  runtime(std::move(rt)),
-                                                  mapperRegistry(std::make_shared<MapperRegistry>()),
-                                                  eventHandlerRegistry(std::make_shared<EventHandlerRegistry>()),
+                                               PlatformDepMethodsHolder platformDepMethodsHolder) :
+                                                  NativeReanimatedModuleSpec(jsInvoker),
+                                                  RuntimeManager(std::move(rt), errorHandler, scheduler),
+                                                  mapperRegistry(new MapperRegistry()),
+                                                  eventHandlerRegistry(new EventHandlerRegistry()),
```

Comment:
Why you use here `new` instead of `make_shared`? I prefer `make_shared` because is safer and allocating memory only once.

Reply:
Fixed in https://github.com/software-mansion/react-native-reanimated/pull/1861/commits/b638ad7bdcfbb53ce5535234c5fb41139709f89e